setup.py

```diff
@@ -44,7 +44,7 @@
     install_requires=[
         "astropy>=4,<5",
         "bokeh~=1.0",
-        "eventio>=1.1.1,<2.0.0a0",  # at least 1.1.1, but not 2
+        "eventio>=1.5.0,<2.0.0a0",  # at least 1.1.1, but not 2
```

Comment:
at least 1.5.0

Reply:
What's your point?

Reply:
Please just remove the comment. It does not any further information.